scripts/test_usb_scan.py

```diff
@@ -2,10 +2,7 @@
 #
 # unittest for usb_scan.py
 
-try:
-    from collections.abc import Mapping, Container, Iterable
-except ImportError: # python2
-    from collections import Mapping, Container, Iterable
+from collections.abc import Mapping, Container, Iterable
```

Comment:
Remove unused import: "Container" and "Iterable" found by pylint:
scripts/test_usb_scan.py:5:0: W0611: Unused Container imported from collections.abc (unused-import)
scripts/test_usb_scan.py:5:0: W0611: Unused Iterable imported from collections.abc (unused-import)